lib/vsc/install/shared_setup.py

```diff
@@ -1552,11 +1552,20 @@ def search_replace(txt):
     def build_setup_cfg_for_bdist_rpm(target):
         """Generates a setup.cfg on a per-target basis.
 
-        Create [bdist_rpm] section with
+        Can be skipped by setting 'makesetupcfg' to False in setup.py
+
+        Creates [bdist_rpm] section with
             install_requires => requires
             provides => provides
             setup_requires => build_requires
 
+        Creates [metadata] section with
+            description-file => README file
+
+        Creates [install] section if needed,
```

Comment:
only when building rpm?

Reply:
No, only when `install-scripts` is specified in `setup.py`.

Or I don't understand what you want me to change :)

Reply:
the is called in a method called build_setup_cfg_for_bdist_rpm, i'm going to assume that only rus when building rpms?

Reply:
Yes, and that's fine. We only need to control the location of the scripts when building an RPM, not when installing via `setup.py install`.